app/services/metrics/metrics_processor.rb

```diff
@@ -0,0 +1,73 @@
+module Metrics
+  ##
+  # Does the processing of the Github events previously stored and leaves the
+  # generated metrics in the Metrics table.
+  #
+  # It takes care of processing only the metrics that are pending to be
+  # processed.
+  class MetricsProcessor < BaseService
```

Comment:
I find this class name confusing because there is also a  `BaseMetricProcessor` and the rest of the processor implementations